src/main/resources/db/migration/all/20230628104650__add_arrival_date_time_to_arrivals.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE arrivals ADD COLUMN arrival_date_time TIMESTAMP WITH TIME ZONE;
+
+UPDATE arrivals SET arrival_date_time = cast(arrival_date as timestamp) at time zone 'utc';
```

Comment:
I guess my thinking was that all the existing arrivals would be CAS3 arrivals where the time is less important. This would also allow us to delete the duplicated column once we're confident it's all working